source/manuals/monitor-paas-app-with-prometheus.html.md.erb

```diff
@@ -33,12 +33,23 @@ You can create a Prometheus service within your PaaS space and allow it to bind
   - `cf create-service gds-prometheus prometheus <service-instance-name>`
 2. Either [update your app's manifest.yml] to bind your new service or bind it using the CLI
   - `cf bind-service <app-name> <service-instance-name>`
+3. If your app uses a custom domain, you need to make sure that the [authorization header][] is being forwarded otherwise the basic auth to the metrics endpoint will fail.
+  - `cf update-service <cdn-route-service> \
+    -c '{"domain": "<custom-domain>", "headers": ["Accept", "Authorization"]}'`
```

Comment:
maybe make clear that if the user has any other headers that they need to forward, they need to be listed here too?

Reply:
ok